### PR TITLE
fix (PatientHome): changed no medical data condition

### LINKED ISSUE #3132: Patient Malfunction
image

### PR DESCRIPTION
**Pull request by khavinshankar**
Fixes #3132

### CODE REVIEW

File: src/Components/Patient/PatientHome.tsx [hunk 1]
```diff
@@ -1031,11 +1031,11 @@ export const PatientHome = (props: any) => {
               <div className="border-b border-dashed text-gray-900 font-semibold text-center text-lg pb-2">
                 Medical
               </div>
-              {/* No medical data found */}
               {!patientData.present_health &&
                 !patientData.allergies &&
                 !patientData.ongoing_medication &&
-                !patientData.is_antenatal && (
+                !patientData.is_antenatal && 
+                !patientData.medical_history?.length (
```
Comment (khavinshankar) [inline src/Components/Patient/PatientHome.tsx]:
```suggestion
                !patientData.medical_history?.length && (
```

**Comment by nihal467** [comment]
image

@khavinshankar make the grey, more darker

**Comment by nihal467** [comment]
@khavinshankar can you merge the latest care version and change the label to need testing

**Comment by nihal467** [comment]
test approved

File: src/Components/Patient/PatientHome.tsx [hunk 2]
```diff
@@ -1062,12 +1054,13 @@ export const PatientHome = (props: any) => {
...
                 Medical
               </div>
-              {/* No medical data found */}
               {!patientData.present_health &&
                 !patientData.allergies &&
                 !patientData.ongoing_medication &&
-                patientData.gender === 2 &&
-                !patientData.is_antenatal && (
+                !(patientData.gender === 2 && patientData.is_antenatal) &&
+                !patientData.medical_history?.filter(
```
Comment (gigincg) [inline src/Components/Patient/PatientHome.tsx]: You could've used array.some here right?

**Review by gigincg — APPROVED** [review]

**Pull request merged**